Sprint-2/2-mandatory-debug/2.js

```diff
@@ -13,12 +13,22 @@ console.log(`The last digit of 42 is ${getLastDigit(42)}`);
 console.log(`The last digit of 105 is ${getLastDigit(105)}`);
 console.log(`The last digit of 806 is ${getLastDigit(806)}`);
 
-// Now run the code and compare the output to your prediction
-// =============> write the output here
-// Explain why the output is the way it is
-// =============> write your explanation here
+// Now run the code and compare the output to your prediction 
+// - 3 - slice (-1) takes from const number and not from getLastDigit
+// Explain why the output is the way it is - in the code don't getLastDigit parameter
+// in the code don't getLastDigit parameter
 // Finally, correct the code to fix the problem
 // =============> write your new code here
 
+const num = 103;
+
```

Comment:
This second declaration of `num` means this file can't be executed successfully.